v2/spanner-common/terraform/samples/mysql-vpc-spanner/outputs.tf

```diff
@@ -0,0 +1,25 @@
+output "resource_ids" {
+  description = "IDs of resources created during set up."
+  value = {
+    network          = google_compute_network.vpc_network.name
+    subnetwork       = google_compute_subnetwork.vpc_subnetwork.name
+    mysql_db_ip      = google_compute_instance.mysql_database_instance.network_interface[0].network_ip
+    spanner_instance = google_spanner_instance.spanner_instance.name
+  }
+}
+
+output "resource_urls" {
```

Comment:
should we give some monitoring links as well?

Reply:
This is intended to create a test stack for MySQL inside a VPC reproduce and test features around VPC peering. 
Monitoring is not relevant here.